R/produce_source_sc_tests.R

```diff
@@ -0,0 +1,51 @@
+#' Source Care Home Tests
+#'
+#' @description Produce a set of tests which can be used by Care Home and Home Care
+#' This will produce counts of various demographics
+#' using [create_demog_test_flags()] counts of episodes for every `hbtreatcode`
+#' using [create_lca_test_flags()]
+#' It will also produce various summary statistics for bedday, cost and
+#' episode date variables.
+#'
+#' @param data new or old data for testing summary flags
+#' (data is from [get_source_extract_path()])
+#' @param sum_mean_vars variables used when selecting 'all' measures from [calculate_measures()]
+#' @param max_min_vars variables used when selecting 'min-max' from [calculate_measures()]
+#'
+#' @return a dataframe with a count of each flag
+#' from [calculate_measures()]
+#' @export
+#'
+#' @family extract test functions
+#' @seealso calculate_measures
+produce_source_sc_tests <- function(data,
+                                    sum_mean_vars = c("beddays", "cost", "yearstay"),
+                                    max_min_vars = c(
+                                      "record_keydate1", "record_keydate2",
+                                      "cost_total_net", "yearstay"
```

Comment:
```suggestion
                                    sum_mean_vars = c("beddays", "cost", "yearstay", "hours"),
                                    max_min_vars = c(
                                      "record_keydate1", "record_keydate2",
                                      "cost_total_net", "yearstay", "hours"
```
Adding `hours` into this block of code will pick up the max/min and the mean of HC hours for Q1-4